tests/integration-tests/tests/dashboard/test_dashboard/test_dashboard/pcluster.config.yaml

```diff
@@ -0,0 +1,26 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+Scheduling:
+{% if {{ scheduler }} == "awsbatch" %}Awsbatch:{% else %}Slurm:{% endif %}  # FIXME: need to make it camelcase
+    Queues:
+      - Name: queue1
+        Networking:
+          SubnetIds:
+            - {{ public_subnet_id }}
+        ComputeResources:
+          - Name: compute-resource1
+            InstanceType: {{ instance }}
+Monitoring:
+  DetailedMonitoring: true
```

Comment:
Why DetailedMonitoring is true?

Reply:
I think we haven't test DetailedMonitoring else where. It would be better to have a test cover it to make sure it work. So I set DetailedMonitoring=true here to make sure it is covered. If we are plan to test it somewhere else, then we can set it to false here.